addons/sale/wizard/res_config_settings.py

```diff
@@ -64,12 +64,12 @@ class ResConfigSettings(models.TransientModel):
     module_delivery_dhl = fields.Boolean("DHL Express Connector")
     module_delivery_easypost = fields.Boolean("Easypost Connector")
     module_delivery_envia = fields.Boolean("Envia.com Connector")
-    module_delivery_fedex = fields.Boolean("FedEx Connector")
+    module_delivery_fedex_rest = fields.Boolean("FedEx Connector")
```

Comment:
Maybe silly (and late, definitely non-blocking) question but why don't you rename the modules instead?
Wouldn't it make more sense to have `delivery_fedex` as the reference (REST) and the other one renamed in 19?